tools/make-readmes.js

```diff
@@ -109,6 +109,12 @@ const GETTING_STARTED_END_COMMENT = '<!-- getting started end -->';
         );
         description = JSON.parse(packageJson).description;
 
+        if (
+          packageJson.includes('dotenv -e .env -- autoblocks prompts generate')
```

Comment:
this doesn't handle python, maybe we can make it something you leave as a comment in the READMEs?

Reply:
Well for python it won't ready from .env right?

Reply:
if you want to punt on this you can just add a step to the READMEs you just added in this PR, that's what i've been doing for now: https://github.com/autoblocksai/autoblocks-examples/tree/main/Python/prompt-sdk-headless#set-your-api-key

Reply:
yeah we read from .env for python: https://github.com/autoblocksai/autoblocks-examples/blob/7c66d51d05ef8d2d7861ccab04ec5e700d46c48d/Python/prompt-sdk-headless/src/main.py#L10

Reply:
The poetry install step is what needs it and I don't think that will read from .env so it needs to be set on terminal

Reply:
but for code generation we don't, so probably easiest to just have a separate section where you export the env var since it'll work for both

Reply:
Oh ya sorry I meant the code generation.

Reply:
ok it's fine we can leave it, we can come back and clean up the python side later. can you just fix the missing `>` and give it a different name than the open api key